projects/common/de10nano/de10nano_system_qsys.tcl

```diff
@@ -14,8 +14,9 @@ set_interface_property sys_rst EXPORT_OF sys_clk.clk_in_reset
 
 add_instance sys_hps altera_hps
 set_instance_parameter_value sys_hps {MPU_EVENTS_Enable} {0}
-set_instance_parameter_value sys_hps {F2SDRAM_Type} {AXI-3}
-set_instance_parameter_value sys_hps {F2SDRAM_Width} {64}
+set_instance_parameter_value sys_hps {F2SCLK_SDRAMCLK_Enable} {0}
```

Comment:
I think this line is redundant, it's disabled by default.

Reply:
Yes, you are right.